<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BillView</class>
 <widget class="QWidget" name="BillView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>558</width>
    <height>853</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>账单页</string>
  </property>
  <property name="windowIcon">
   <iconset resource="logo.qrc">
    <normaloff>:/贴图库/logo.png</normaloff>:/贴图库/logo.png</iconset>
  </property>
  <widget class="QLabel" name="lb_fare">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>371</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>显示费用</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pay">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>530</y>
     <width>171</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>支付</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_time">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>321</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>显示时间</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="logo.qrc"/>
 </resources>
 <connections/>
</ui>
